src/slack/slack.js

```diff
@@ -177,11 +177,15 @@ export class Slack {
         const order = this.orders[userId]
 
         if (order) {
+          // avoid multiple clicks in one step
```

Comment:
btw, have you considered the change I proposed, regarding responding with `loading...` text? maybe it makes sense to do it as well? at least on submit, so user has some feedback the bot works on the request

Reply:
I did not use respond, because there occurred cases where `next actions/UI` appeared before `respond('loading')`.